corgidrp/l2a_to_l2b.py

```diff
@@ -61,24 +61,30 @@ def add_shot_noise_to_err(input_dataset, kgain, detector_params):
     return phot_noise_dataset
 
 
-def dark_subtraction(input_dataset, dark, detector_regions=None, outputdir=None):
+def dark_subtraction(input_dataset, noisemaps=None, dark=None, detector_regions=None, outputdir=None):
```

Comment:
why do we need to pass in both? It seems to me that we can only use one regardless.

Reply:
Ah yeah, you're right.